com.unity.probuilder/Editor/MenuActions/Export/Export.cs

```diff
@@ -84,12 +85,23 @@ protected override void OnSettingsGUI()
 
             EditorGUI.BeginChangeCheck();
 
+            EditorGUI.BeginChangeCheck();
```

Comment:
one too many change checks ?

Reply:
There's another `EndChangeCheck` near the bottom of this function